src/argilla/client/feedback/training/frameworks/openai.py

```diff
@@ -34,7 +31,7 @@ def __init__(self, *args, **kwargs):
             self._logger.warning("Seed is not supported for OpenAI. Ignoring seed for training.")
 
         if self._model is None:
-            self._model = "curie"
+            self._model = "gpt-3.5-turbo-0613"
```

Comment:
If we are setting `self._model` if not provided, then we should display a warning to let the user know which model is being used.